driver/controller_internal_test.go

```diff
@@ -480,3 +480,72 @@ func TestIsValidStorageUUID(t *testing.T) {
 	assert.True(t, isValidUUID("1160ffc3-58ec-4670-bdc9-27fe385d281d"))
 	assert.True(t, isValidUUID("0160ffc3-58ec-4670-bdc9-27fe385d281d"))
 }
+
+func TestDriver_CreateSnapshot(t *testing.T) {
+	t.Parallel()
+
+	type args struct {
+		req          *csi.CreateSnapshotRequest
+		volExists    bool
+		volBackingUp bool
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *csi.CreateSnapshotResponse
+		wantErr bool
+	}{
```

Comment:
More tests the better! Maybe one with `wantErr: true` would be useful as well

Reply:
Just added 2 of those!